bin/templates/cordova/lib/prepare.js

```diff
@@ -43,8 +43,16 @@ module.exports.prepare = function (cordovaProject, options) {
 
     this._config = updateConfigFilesFrom(cordovaProject.projectConfig, munger, this.locations);
 
+    // Prepare the min and target SDK version
+    const minSdkVersion = this._config.getPreference('android-minSdkVersion', 'android');
+    const targetSdkVersion = this._config.getPreference('android-targetSdkVersion', 'android');
+
+    let gradlePropertiesUserConfig = {};
+    if (minSdkVersion) gradlePropertiesUserConfig.minSdkVersion = minSdkVersion;
+    if (targetSdkVersion) gradlePropertiesUserConfig.targetSdkVersion = targetSdkVersion;
```

Comment:
Again, please confirm if `gradlePropertiesUserConfig.targetSdkVersion` is the correct property name to be setting or if it should be prefixed with `cdv`. Reference from [Google Android Developer Docs](https://developer.android.com/studio/publish/versioning#minsdkversion).

In @dpogue commit, he does not use `targetSdkVersion`. He uses `cdvCompileSdkVersion`. https://github.com/dpogue/cordova-android/commit/d18f8d18a10a50084130300d5edbb84901b59707

Reply:
> He uses `cdvCompileSdkVersion`. [dpogue@d18f8d1](https://github.com/dpogue/cordova-android/commit/d18f8d18a10a50084130300d5edbb84901b59707)

agreed on my part